src/utils/map-controller.js

```diff
@@ -176,12 +176,14 @@ export default class MapController {
     this.onViewportChange = onViewportChange;
     this.onStateChange = onStateChange;
 
-    if (!this.mapStateProps || this.mapStateProps.height !== options.height) {
+    const dimensionChanged = !this.mapStateProps || this.mapStateProps.height !== options.height;
```

Comment:
Don't need check whether width changed here?

Reply:
Width does not affect normalization. A map may need to be scaled and shifted vertically to fill the height.